src/styles/_catalog.scss

```diff
@@ -1,22 +1,46 @@
-.catalog {
-  display: flex;
-  flex-wrap: wrap;
+$card_width: 200px;
 
+.catalog {
   max-width: 944px;
   margin: 0 auto;
+  display: grid;
+  grid-gap: 48px;
+  justify-items: center;
+  justify-content: center;
```

Comment:
it's redundant

Reply:
I removed "justify-content: center"
and if I remove "justify-items: center;" -> "Cards not centered."